.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 npm-debug.log
+_css
+_temp
```

Comment:
Ignoring the `_css` and `_temp` directories. Why?

Because we'll write our styles in `_scss`, which get compiled into `_temp/style.css`, which then get autoprefixed to `_css/style.css`. This means that the _css directory is auto-built, so there's no sense keeping it under version control.